.github/workflows/combine-xcframework.yml

```diff
@@ -76,12 +76,11 @@ jobs:
 
       - name: Create XCFramework
         run: |
-          xcodebuild -create-xcframework \
-            -library ./device-static/libdevice.a \
-            -headers ./device/jdk/include \
-            -library ./sim-static/libsim.a \
-            -headers ./simulator/jdk/include \
-            -output ./OpenJDK.xcframework
+          chmod +x .github/scripts/build-xcframework.sh
```

Comment:
@abhinayagarwal I would have expected more to be removed from the yml. (like all the steps with libtool which are now part of the script)